sdk/storage/azure-storage-blob/tests/perfstress_tests/_test_base.py

```diff
@@ -51,6 +72,9 @@ def add_arguments(parser):
         parser.add_argument('--max-concurrency', nargs='?', type=int, help='Maximum number of concurrent threads used for data transfer. Defaults to 1', default=1)
         parser.add_argument('-s', '--size', nargs='?', type=int, help='Size of data to transfer.  Default is 10240.', default=10240)
         parser.add_argument('--no-client-share', action='store_true', help='Create one ServiceClient per test instance.  Default is to share a single ServiceClient.', default=False)
+        parser.add_argument(
+            "--use-entra-id", action="store_true", help="Use Microsoft Entra ID authentication instead of connection string."
```

Comment:
Let's also add this to perfstress_tests/README.md file (probably under `Common Blob command line options`?)